eskip/eskip_test.go

```diff
@@ -620,6 +628,15 @@ func TestEditorPreProcessor(t *testing.T) {
 			},
 			routes: r1Filter,
 			want:   r1FilterChanged,
+		},
+		{
+			name: "test that regex with / works properly",
+			rep: &Editor{
+				reg:  regexp.MustCompile("/26"),
+				repl: "/24",
+			},
+			routes: r2,
+			want:   r2Changed,
```

Comment:
I am not sure we need this test - it tests that regexp replacement works but it does not test the change this PR introduces